src/category_theory/limits/shapes/kernels.lean

```diff
@@ -218,26 +218,37 @@ lemma kernel_not_epi_of_nonzero (w : f ≠ 0) : ¬epi (kernel.ι f) :=
 lemma kernel_not_iso_of_nonzero (w : f ≠ 0) : (is_iso (kernel.ι f)) → false :=
 λ I, kernel_not_epi_of_nonzero w $ by { resetI, apply_instance }
 
+-- TODO the remainder of this section has obvious generalizations to `has_equalizer f g`.
+
+instance has_kernel_comp_mono {X Y Z : C} (f : X ⟶ Y) [has_kernel f] (g : Y ⟶ Z) [mono g] :
+  has_kernel (f ≫ g) :=
+{ exists_limit := ⟨{
+  cone := kernel_fork.of_ι (kernel.ι f) (by simp),
```

Comment:
Is this the mathlib style for these things? I thought
```suggestion
{ exists_limit := 
  ⟨{ cone := kernel_fork.of_ι (kernel.ι f) (by simp),
```
was preferred

Reply:
Yes!